server/src/tests/integration.rs

```diff
@@ -435,13 +436,47 @@ fn run_op() -> Result<(Child, Child, Child)> {
 async fn use_optimism_bridge() -> Result<()> {
     pause(Some(Duration::from_secs(OP_START_IN_SECS)));
 
-    deposit_to_l2().await?;
+    deposit_eth_to_l2().await?;
+
+    deposit_erc20_to_l2().await?;
+
     withdrawal::withdraw_to_l1().await?;
 
     Ok(())
 }
 
-async fn deposit_to_l2() -> Result<()> {
+async fn deposit_erc20_to_l2() -> Result<()> {
+    let l1_rpc = var("L1_RPC_URL").unwrap();
+    let from_wallet = get_prefunded_wallet().await?;
+    let receiver = from_wallet.address();
+    let deposit_amount = U256::from(1234);
+
+    // Deploy ERC-20 token to bridge
+    let l1_address = erc20::deploy_l1_token(&from_wallet, &l1_rpc).await?;
+    // Create corresponding token on L2
+    let l2_address = erc20::deploy_l2_token(&from_wallet, l1_address, L2_RPC_URL).await?;
+    // Perform deposit
+    erc20::deposit_l1_token(
+        &from_wallet,
+        l1_address,
+        l2_address,
+        deposit_amount,
```

Comment:
nit: If we rename `deposit_amount` to `amount` this func call fits into a single line.

Reply:
Done in https://github.com/MovedNetwork/op-move/pull/307/commits/3de586aa4b6401bee698efda703acb6428e7379d